transforms/code/license_copyright_removal/ray/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM docker.io/rayproject/ray:2.9.3-py310
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Install system dependencies, including libgomp1
+RUN sudo apt-get update && sudo apt-get install -y \
+    libgomp1 \
+    && sudo rm -rf /var/lib/apt/lists/*
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
+RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
+COPY --chown=ray:users data-processing-lib-ray/ data-processing-lib-ray/ 
+RUN cd data-processing-lib-ray    && pip install --no-cache-dir -e .
+COPY --chown=ray:users python-transform/ python-transform 
+RUN cd python-transform && pip install --no-cache-dir -e .
+
+COPY requirements.txt requirements.txt
```

Comment:
Remove this install of requirements.txt, the pyproject.toml replaces this and should define the requirements